server/controllers/personalInfo.js

```diff
@@ -3,17 +3,30 @@ const asyncHandler = require("express-async-handler");
 
 exports.createPersonalInfo = asyncHandler(async (req, res) => {
   try {
-    const info = PersonalInfo.create({
-      userId: req.user.id,
-      firstName: req.body.firstName,
-      middleInit: req.body.middleInit,
-      lastName: req.body.lastName,
-      email: req.body.email,
-      phone: req.body.phone,
-      birthday: req.body.birthday,
-      gender: req.body.gender,
-      about: req.body.about
+    const hasSubmitted = await PersonalInfo.findOne({
+      user: req.user.id
     });
+    if (hasSubmitted) {
+      hasSubmitted = {...req.body};
```

Comment:
Won't spreading the `req.body` into `hasSubmitted` replace the entire object? (The document will no longer contain the userId in this case)